commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/Complex.java

```diff
@@ -2904,7 +2782,8 @@ private static Complex atanh(final double real, final double imaginary,
      * @param y the y value
      * @return {@code x^2 + y^2 - 1}.
      */
-    private static double x2y2m1(double x, double y) {
+    //TODO - make it private in future
```

Comment:
Since the intent is to eventually move all these private functions to ComplexFunctions, then please start the process now. Make any function that is currently also required in Complex package-private and add the TODO note to make private in CompexFunctions in the future.

It may be convenient to leave the functions locally as pass through functions:
```java
private static double x2y2m1(double x, double y) {
    return ComplexFunctions.x2y2m1(x, y);
}
```
This will prevent having to update all the call sites in Complex and minimise the diff.